Dockerfile

```diff
@@ -0,0 +1,43 @@
+# Dockerfile to build and test DeepDive inside a container
+#
+# `make build-in-container test-in-container` uses master image built by this.
+# `util/build/docker/` contains utilities relevant to this.
+FROM ubuntu
```

Comment:
And yes, `ubuntu` is `ubuntu:latest` and I have to check if their policy is to strictly follow LTS or literally any latest release. (I hope the former but may not be the case)